x-pack/plugins/security_solution/common/endpoint/data_generators/endpoint_action_generator.ts

```diff
@@ -76,6 +74,14 @@ export class EndpointActionGenerator extends BaseDataGenerator {
   ): LogsEndpointActionResponse {
     const timeStamp = overrides['@timestamp'] ? new Date(overrides['@timestamp']) : new Date();
 
+    const startedAtTimes = [2, 3, 5, 8, 13, 21].reduce<number[]>((acc, curr) => {
```

Comment:
For readability, could we write this with a `map` instead of `reduce`? Same for the one in fleet generator.

Reply:
dd244ae3f28b5a2945b139e9f2f21344e1a4c7a9